rules-reviewed/camel3/camel2/xml-renamed-components.windup.xml

```diff
@@ -62,6 +62,23 @@
                 <matches pattern="([a-zA-Z0-9]|-)*" />
             </where>
         </rule>
+        <rule id="xml-renamed-components-00003">
+            <when>
+                <project>
+                    <artifact groupId="org.apache.camel" artifactId="camel-mongodb3" />
+                </project>
+            </when>
+            <perform>
+                <hint title="`org.apache.camel:camel-mongodb3` artifact has been renamed to `org.apache.camel:camel-mongodb`" effort="1" category-id="mandatory" >
```

Comment:
to keep the title consistent with the previous rules in the same file i think the title should change a little bit
```suggestion
                <hint title="`org.apache.camel:camel-mongodb3` artifact has been renamed" effort="1" category-id="mandatory" >
```